tap_jira/streams.py

```diff
@@ -369,3 +379,7 @@ def validate_dependencies():
             errs.append(msg_tmpl.format("Issue Transitions", "Issues"))
     if errs:
         raise DependencyException(" ".join(errs))
+
+def transform_user_date(user_date):
+    """Transform date value to 'yyyy-mm-dd' format."""
+    return dateparser.parse(user_date).strftime('%Y-%m-%d')
```

Comment:
We have to check the locale and change the format. US is MM/DD/YY and this change will cause issues

Reply:
API returns `userReleaseDate` and `userStartDate` always in the `dd/mm/yyyy` format where the month name is in Abbreviation form. `Dateparser` library itself handles locale value and converts Abbreviation month to number. For example, if userReleaseDate is `12/abr/2022` then we are converting it to `2022-04-12`.Then, at the end singer-python will transform any DateTime to `%Y-%m-%dT00:00:00Z` format.